ApplyToTheIndustry/Assets/Scripts/ResumeBuilder/SubmitButton.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SubmitButton : MonoBehaviour
```

Comment:
Ah, I see you replaced the null action with this. I like it, it might be more extensible. I think it can also be used for the course selection and so on.